frontend/ui/widget/inputdialog.lua

```diff
@@ -961,15 +966,17 @@ function InputDialog:_addScrollButtons(nav_bar)
             })
             -- Add a button to go to the line by its number in the file
             table.insert(row, {
-                text = _("Go"),
+                text = "", -- current line number
+                font_bold = false,
+                id = "go",
                 callback = function()
                     self:toggleKeyboard(false) -- hide text editor keyboard
-                    local cur_line_num, last_line_num = self._input_widget:getLineNums()
+                    local curr_line_num, last_line_num = self._input_widget:getLineNums()
```

Comment:
Not certain we have consistency and that one is preferable to the other:
```
$ rgrep ' cur_' frontend plugins  | wc -l
270

$ rgrep ' curr_' frontend plugins  | wc -l
124
```

Reply:
I'd personally stick with `cur_` but it doesn't seem very important (unless it's too close to each other).